packages/core/src/index.ts

```diff
@@ -176,13 +176,14 @@ export const diagram = async (
     variation: string;
   },
   node: HTMLElement,
-  pathResolver: PathResolver
+  pathResolver: PathResolver,
+  name: string
```

Comment:
To make this a non-breaking change, we need to make this `name` parameter optional.